internal/operator/namespace.go

```diff
@@ -32,21 +32,21 @@ func CreateNamespace(ctx context.Context, name string) (*corev1.Namespace, error
 	}
 	_, err := operatorClient.CoreV1().Namespaces().Create(ctx, &nsSpec, metav1.CreateOptions{})
 	if err != nil {
-		logger.Errorf("error while creating Namespace %s: %s", name, err)
+		logger.Errorf("error while creating Namespace %s: %w", name, err)
 		return nil, err
 	}
-	logger.Debugf("Namespace Created: %s", name)
+	logger.Debugw("Namespace Created", name)
```

Comment:
Debugf? If using Debugw need to fix format; (msg string, keysAndValues ...interface{})